packages/dds/cell/src/cell.ts

```diff
@@ -250,15 +254,19 @@ export class SharedCell<T = any> extends SharedObject<ISharedCellEvents<T>>
      * For messages from a remote client, this will be undefined.
      */
     protected processCore(message: ISequencedDocumentMessage, local: boolean, localOpMetadata: unknown) {
+        const _localOpMetadata = localOpMetadata as ICellLocalOpMetadata;
```

Comment:
Since the input localOpMetadata is set as unknown, I make an additional internal casting. But I did not find anything in official doc about naming convention after type casting, so I just used underscore here. Is that good enough or any suggestions? @scarlettjlee

Reply:
I'm okay with that. You could also just name it something else like "cellOpMetadata".